lxd/db/generate/db/method.go

```diff
@@ -792,7 +792,12 @@ func (m *Method) create(buf *file.Buffer, replace bool) error {
 			m.ifErrNotNil(buf, true, "-1", fmt.Sprintf(`fmt.Errorf("Failed to get \"%s\" prepared statement: %%w", err)`, stmtCodeVar(m.entity, kind)))
 			buf.L("// Execute the statement. ")
 			buf.L("result, err := stmt.Exec(args...)")
-			m.ifErrNotNil(buf, true, "-1", fmt.Sprintf(`fmt.Errorf("Failed to create \"%s\" entry: %%w", err)`, entityTable(m.entity, m.config["table"])))
+			buf.L("if err != nil {")
+			buf.L("if query.IsConflictErr(err) {")
+			buf.L(`return -1, api.StatusErrorf(http.StatusConflict, "This \"%s\" entry already exists")`, entityTable(m.entity, m.config["table"]))
+			buf.L("}\n")
+			buf.L(`return -1, fmt.Errorf("Failed to create \"%s\" entry: %%w", err)`, entityTable(m.entity, m.config["table"]))
```

Comment:
For consistency, the "Failed to create" should be reordered to align with the rename and update failure messages.